system/jlib/jutil.hpp

```diff
@@ -360,10 +360,18 @@ extern jlib_decl bool getConfigurationDirectory(const IPropertyTree *dirtree, //
                                                 const char *instance, 
                                                 StringBuffer &dirout);
 
-extern jlib_decl bool querySecuritySettings(bool *          _useSSL,
+extern jlib_decl bool querySecuritySettings(SSLCfg *        _useSSL,
                                             unsigned short *_port,
                                             const char * *  _certificate,
-                                            const char * *  _privateKey);
+                                            const char * *  _privateKey,
+                                            const char * *  _passPhrase);
+
+extern jlib_decl bool queryDafsServerSecSettings(SSLCfg *        _useSSL,
+                                                 unsigned short *_port,
+                                                 unsigned short *_sslport,
+                                                 const char * *  _certificate,
+                                                 const char * *  _privateKey,
+                                                 const char * *  _passPhrase);
```

Comment:
I think this (and the code that used it) would be a lot clearer/better if filled in a structure
```
struct DaFSSettings
{
  unsigned short port = 0, sslPort = 0;
  StringAttr certificate, privateKeyFile, passPhrase;
}
bool getDaFsSettings(DaFSSettings &dafsInfo)
{
}
```